### ISSUE TITLE
API key? 

### ISSUE THREAD

**Issue by floswald** [phamtrisi/airapi#8]
hi @phamtrisi 
thanks for sharing this code! i have a quick question: you hard code an API key in here. how did you obtain that API key? i am planning to make a very large number of requests against that API, so I am worried that there might be a limit at some point? do you have any experience with such issues? thanks for your help.

**Comment by JJ** [comment]
That's correct. Besides, the username and password in the example do not seem to work, since at the end of the day the code uses the API key. There should be some way to configure it, too.
